src/Console/Command/ReadmeCommand.php

```diff
@@ -131,6 +131,15 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
     $ brew install php-cs-fixer
 
+Locally (PHIVE)
+~~~~~~~~~~~~~~~~~~~
```

Comment:
Why so long? Other underline-ish lines are with the length of the text above.

Reply:
I'll admit, the lengthy `~` is a result of copying another (sub)header, I'll update soon, thanks for the sharp review :+1:   :)